CO-3049 Muskathlon migration v11

josueBulle opened the pull request:
Merge first:

website_event_compassion
quentin/partner_compassion
quentin/sponsorship_switzerland
quentin/report_compassion
quentin/child_sync_wp
quentin/sms_939
quentin/ogone
quentin/child_switzerland
quentin/partner_communication

Various fixes on [compassion_modules](https://github.com/CompassionCH/compassion-modules/pull/1288)

ecino commented on muskathlon/models/advocate_details.py:
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
You should make inherit the class from GeoModel

ecino reviewed (changes requested):
```
muskathlon/controllers/main.py:10:22: E401 multiple imports on one line
```
Can you rebase your branch onto 11.0 ? I think some travis errors are because you have an old version.

josueBulle committed:
CO-3049

Added geo_model inheritance in advocate_details

josueBulle commented:
I don't understand Travis errors...
There are a lot of "ConnectionRefusedError: [Errno 111] Connection refused"

ecino commented:
> I don't understand Travis errors...
> There are a lot of "ConnectionRefusedError: [Errno 111] Connection refused"

Your branch is behind v11. It runs the tests of the other modules which are in an older version and I think we had some mistakes previously. Maybe it will fix the errors if you rebase properly so that you commits sit on top of the current 11.0 state.

![image](https://user-images.githubusercontent.com/8435180/75360192-c5dad800-58b5-11ea-97c1-aa2d5f35925a.png)

josueBulle commented:
I did rebase when you asked yesterday. In your screenshot, it is 10 commits behind v10. It is up to date with branch v11.
Muskathlon has no test... Idk what I did/understood wrong

josueBulle committed:
Fix sms939 and crm_switzerland migrations

- don't catch error, don't lower the timeout in sms939 tests
- correct function name
- lints
a

josueBulle commented:
I included two fixes for already-merged migrations instead of creating a new PR
